### PR TITLE
Graph api and Az Storage changes

### PR DESCRIPTION
**Pull request by Akreddy123**
## Description
</br>
 Brief description of the requested change.

## Checklist
- [ ] I have read the instructions mentioned in [Contribute to Code](/Contributing.md).
- [ ] I have read and understood the criteria described under [submitting changes](/Contributing.md#submitting-changes). 
- [ ] The title of the PR clearly describes the intent of the PR.
- [ ] This PR does not introduce any breaking changes to the code.

### CODE REVIEW

File: src/AzSK.Framework/Helpers/Helpers.ps1 [hunk 1]
```diff
@@ -433,51 +433,39 @@ class Helpers {
     static [string] GetAccessToken([string] $resourceAppIdUri, [string] $tenantId) 
     {
         $rmContext = [Helpers]::GetCurrentRMContext()
```
Comment (v-zhgup) [inline src/AzSK.Framework/Helpers/Helpers.ps1]: I think we should change the method and variable name here, its confusing, sounds like AzureRm.

Comment (Akreddy123) [reply src/AzSK.Framework/Helpers/Helpers.ps1]: that method is heavily used in our module. will do that in the future PRs

File: src/AzSK/AzSK.psm1 [hunk 2]
```diff
@@ -496,6 +496,27 @@ function Clear-AzSKSessionState {
 
 }
 
+function Set-AzSKStorageBlobContent([string] $fileName, [string] $blobName, [string] $containerName, [object] $stgCtx)
```
Comment (VishalHaibatpure) [inline src/AzSK/AzSK.psm1]: What extra feature this command is providing against using Set-AzStorageBlobContent

File: src/AzSK/AzSK.psm1 [hunk 3]
```diff
@@ -496,6 +496,27 @@ function Clear-AzSKSessionState {
...
+            $blob = $stgCtx.StorageAccount.CreateCloudBlobClient().GetContainerReference($containerName).GetBlockBlobReference($blobName)
+            $task = $blob.UploadFromFileAsync($fileName)
+            $task.Wait()
+            if ($task.IsCompleted -and !$task.IsFaulted)
+            {
+               Write-Host "Transfer succeeded!"
+            }
+            else
+            {
+                Write-Host "Transfer failed!! Retry?"
```
Comment (VishalHaibatpure) [inline src/AzSK/AzSK.psm1]: statement contains Retry?... are we trying to get input from user?

**Pull request merged**